test/extras/run.py

```diff
@@ -23,3 +25,17 @@
             ]
         )
         sub_process.check_returncode()
+        ocsp_cache_dir_path = pathlib.Path(
+            snowflake.connector.ocsp_snowflake.OCSP_RESPONSE_VALIDATION_CACHE.file_path
+        ).parent
+        cache_files = set(os.listdir(ocsp_cache_dir_path))
+        # This is to test SNOW-79940, making sure tmp files are removed
+        # Windows does not have ocsp_response_validation_cache.lock
```

Comment:
Let's explicitly detect the platform and assert separately for windows in an if case.

Reply:
smart, I will make the change